docs/GettingStarted/configuration/ExporterCommittime.md

```diff
@@ -13,70 +22,223 @@ Currently we support GitHub, GitLab, BitBucket, Gitea and Azure DevOps Git servi
+>
+- [Using Commit Time with Git APIs](#using-commit-time-with-git-apis) for the `git` PROVIDER
+- [Using Commit Time with Images](#using-commit-time-with-images) for the `image` PROVIDER
+
+#### ➔ [PROVIDER](#provider) `git` options
+
+Those options are only applicable to the Commit Time Exporter when the [PROVIDER](#provider) is set to `git` or the [PROVIDER](#provider) option is unset.
+
+| Variable | Required | Default Value |
+|----------|----------|---------------|
+| [API_USER](#api_user) | yes | - |
+| [TOKEN](#token) | yes | - |
+| [GIT_API](#git_api) | yes | [see more...](#git_api) |
+
+
+###### GIT_PROVIDER
+
+- **Required:** no
+    - Only applicable for [PROVIDER](#provider) unset or set to `git`
+    - **Default Value:** github
+- **Type:** string
+
+: Set Git provider type. Can be `github`, `bitbucket`, `gitea`, `azure-devops` or `gitlab`
+
+###### API_USER
+
+- **Required:** yes
+    - Only applicable for [PROVIDER](#provider) set to `git` or unset [PROVIDER](#provider)
+- **Type:** string
+
+: GIT API username.
+
+###### TOKEN
+
+- **Required:** yes
+    - Only applicable for [PROVIDER](#provider) set to `git` or unset [PROVIDER](#provider)
+- **Type:** string
+
+: User's Git API Token
+
+###### GIT_API
+
+- **Required:** yes
+    - Only applicable for [PROVIDER](#provider) set to `git` or unset [PROVIDER](#provider)
+    - **Default Value:**
+        - `api.github.com` for `github` [GIT_PROVIDER](#git_provider)
+        - `dev.azure.com` for `azure-devops` [GIT_PROVIDER](#git_provider)
+        - `try.gitea.io` for `gitea` [GIT_PROVIDER](#git_provider)
+        - **unset** for `gitlab` [GIT_PROVIDER](#git_provider)
+        - **unset** for `bitbucket` [GIT_PROVIDER](#git_provider)
+- **Type:** string
+
+: GitHub, Gitea or Azure DevOps API FQDN. This allows the override for Enterprise users.
+
+#### ➔ [PROVIDER](#provider) `image` options
+
+Those options are only applicable to the Commit Time Exporter when the [PROVIDER](#provider) is set to `image`.
+
+| Variable | Required | Default Value |
+|----------|----------|---------------|
+| [COMMIT_DATE_ANNOTATION](#api_user) | no | `io.openshift.build.commit.date` |
+| [COMMIT_DATE_FORMAT](#token) | no | `%a %b %d %H:%M:%S %Y %z` |
+
+###### COMMIT_DATE_ANNOTATION
+
+- **Required:** no
+    - Only applicable for [PROVIDER](#provider) set to `image`
+    - **Default Value:** io.openshift.build.commit.date
+- **Type:** string
+
+: Annotation name associated with the Image from which commit time is taken. The format of the date and time found in the OpenShift object must match the one from the [COMMIT_DATE_FORMAT](#commit_date_format).
+
+###### COMMIT_DATE_FORMAT
+
+- **Required:** no
+    - Only applicable for [PROVIDER](#provider) set to `image`
+    - **Default Value:** %a %b %d %H:%M:%S %Y %z
+- **Type:** string
+
+: Format in `1989 C standard` to convert time and date found in the Docker Label `io.openshift.build.commit.date` or annotation for the Image.
```

Comment:
just Docker or can be any OCI implementation?

Reply:
changed

Reply:
changed